packages/cli/src/controllers/e2e.controller.ts

```diff
@@ -76,7 +76,6 @@ export class E2EController {
 
 	@Post('/reset')
 	async reset(req: ResetRequest) {
-		config.set('ui.banners.v1.dismissed', true);
```

Comment:
this was added to make e2e tests work, because the banner was preventing clicks on the header bar. 
I'm assuming you've fixed this issue elsewhere in the code ?

Reply:
Yeah, now the banner is part of the layout. It's not overlapping anything.

Reply:
great job switching to a proper grid 🙇🏽 🎉

Reply:
Yeah, this will be the major benefit of this PR, banners just take all the spotlight 😀